contacts/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from datetime import date
+from users.models import Apartment, Seeker
+
+
+class Connection(models.Model):
+    apartment = models.ForeignKey(Apartment, on_delete=models.CASCADE, related_name='apt_connection')
+    seeker = models.ForeignKey(Seeker, on_delete=models.CASCADE, related_name='seeker_connection')
+    date_created = models.DateField(default=date.today)
+
```

Comment:
I think we should add a field like `status` (seeker request -> owner approval/reject) that represents the current status (obviously) of the connection.
That status can help us to filter between the connections and get the relevant data.
It can be an enum or boolean fields.

Reply:
I disagree, I think that the 'pending request' should be a future feature and should not be handled right now, but added in advanced stages of our project (for now - once a seeker pressed 'yes', he is in contact with the apartment). I'm currently following our ERD, but I'll listen to the majority's decision on this one.

Reply:
Please come to a decision on this before I finish my review.

Reply:
It's ok for me to not add the 'status' field in this PR if you we agree that it's should be a future feature.

Reply:
@simzacks We decided to leave it as is, please review :)

Reply:
Sorry for the delay.

I completely agree with @tamirho I think there should be a `status` field for every `connection`. I do not see how it works without.
IMO there should be three options for the `status` field
-Pending: `Seeker` selected `yes` and waiting for `Apartment` response.
-Rejected: `Seeker` selected `yes` and `Apartment` selected `no`. 
-Confirmed: `Seeker` selected `yes` and `Apartment` selected `yes`.

Reply:
Anyway, for now, IMO this PR is ok.
we should discuss about the field and implement (or not) in the next PR.

Reply:
> Anyway, for now, IMO this PR is ok.
> we should discuss about the field and implement (or not) in the next PR.

I agree this should be discussed more deeply, but since most of us voted to leave this part to the future and keep this PR as is - I'll leave it as is. As you said, this can be added in next PR's without a big problem if theres a need.